pkg/maps/ctmap/ctmap.go

```diff
@@ -539,6 +542,15 @@ func (f *GCFilter) doFiltering(srcIP net.IP, dstIP net.IP, dstPort uint16, nextH
 		}
 	}
 
+	if f.MatchIPs != nil {
+		_, srcIPExists := f.MatchIPs[srcIP.String()]
+		_, dstIPExists := f.MatchIPs[dstIP.String()]
+		if srcIPExists || dstIPExists {
+			log.Infof("Removing entry due to match %v %v", srcIPExists, dstIPExists)
```

Comment:
Should we move this to Debug? Those messages can be verbosed for a long-lived endpoint with long-lived connections.

Also, replace `%v` with `%t` and write a better message, `Removing entry due to match true true` doesn't seem useful

Reply:
I've removed it complete. I've used it to verify correctness.